installCatalog.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# use the command line interface to install standard actions deployed
+# automatically
+#
+# To run this command
+# ./installCatalog.sh  <AUTH> <APIHOST> <KAFKA_TRIGGER_HOST> <KAFKA_TRIGGER_PORT>
+# AUTH and APIHOST are found in $HOME/.wskprops
+
+set -e
+set -x
+
+: ${OPENWHISK_HOME:?"OPENWHISK_HOME must be set and non-empty"}
+WSK_CLI="$OPENWHISK_HOME/bin/wsk"
+
+if [ $# -eq 0 ]
+then
+echo "Usage: ./installCatalog.sh <authkey> <apihost> <kafkatriggerhost> <kafkatriggerport>"
```

Comment:
I think this usage should be displayed if if the number if arguments inputted does not equal to four instead of zero.